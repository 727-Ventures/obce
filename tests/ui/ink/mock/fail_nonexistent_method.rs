#[obce::definition]
pub trait Trait {}

#[obce::mock]
impl Trait for () {
    fn method(&self) {}
}

fn main() {}
