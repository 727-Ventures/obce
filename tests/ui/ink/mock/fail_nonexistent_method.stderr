error[E0277]: the trait bound `(dyn Trait + 'static): MethodDescription<3189460888>` is not satisfied
 --> tests/ui/ink/mock/fail_nonexistent_method.rs:4:1
  |
4 | #[obce::mock]
  | ^^^^^^^^^^^^^ the trait `MethodDescription<3189460888>` is not implemented for `(dyn Trait + 'static)`
  |
  = note: this error originates in the attribute macro `obce::mock` (in Nightly builds, run with -Z macro-backtrace for more info)
